HOW_TO.md

```diff
@@ -103,23 +104,48 @@ Once the image is built, you can run it locally or push it to an image repositor
 Please note that there may be naming requirements for pushing images to a repository.
 
 **Testing the Docker image**
```

Comment:
A better title for this might be 'Testing production readiness of Docker image'; hopefully something shorter. This section should be moved below '(OPTIONAL)...' below